microsoft-edge/extensions-chromium/developer-guide/declare-permissions.md

```diff
@@ -89,6 +89,7 @@ The following table lists the currently available permission strings to use in y
 | `scripting` | Gives your extension access to the `chrome.scripting` API. |
 | `search` | Gives your extension access to the `chrome.search` API. |
 | `sessions` | Gives your extension access to the `chrome.sessions` API. |
+| `sidePanel` | Gives your extension access to the `chrome.sidePanel` API.  See [Add a sidebar UI to an extension](../publish/sidebar.md). |
```

Comment:
in file declare-permissions.md
@Bhuvana-Priya - review this row i added, with link

Reply:
newer [comment](https://github.com/MicrosoftDocs/edge-developer/pull/2648/files#r1234535855)